underlay/src/main/resources/config/underlay/cmssynpuf/ui.json

```diff
@@ -1,4 +1,7 @@
 {
+  "featureConfig": {
+    "enableAddByCode": true
```

Comment:
should we enable this for aou_test as well?

Reply:
I don't think it makes sense to duplicate all of the criteria since then they'll diverge from the underlying ones. Measurement was already duplicated though so I added it there.